torchfix/__main__.py

```diff
@@ -14,7 +14,8 @@ def main() -> None:
 
     parser.add_argument(
         "path",
-        nargs="+",
+        nargs="*",
```

Comment:
The current behavior is intentional to mimic ruff's API.

Reply:
`--version` won't work otherwise since the invocation will have to look like `torchfix --version .`

Reply:
For context this is what this looks like if you don't change that:

```
❯ torchfix --version
usage: torchfix [-h] [--fix] [-j JOBS] [--select {ALL}] [--version] [--show-stderr] path [path ...]
torchfix: error: the following arguments are required: path
```

Reply:
Maybe it's a bigger change, but `torchfix --version` should work, and just `torchfix` should print help.

Reply:
I think nargs is fine, but without default?

Reply:
Okay changed!